### DISCUSSION TITLE
Change breadcumb item (text) in Custom Page..

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6273] [Help] [answered]
I create custom page called "Detail" and it's working and even I can change the heading translation, but in the breadcum it is coming like `SomeData/Detail` and I cannot change the "Detail" translation.

Any work around to change this breadcumb item text

![Screenshot from 2023-04-21 12-22-44](https://user-images.githubusercontent.com/28250303/233548046-a148047c-a3ed-497b-bfb5-6dfe6215da7a.png)

**Edited**

I thought in this [URL](https://filamentphp.com/docs/2.x/admin/pages/getting-started#customization) the below line will do what I need, but it doesnot change anything

`protected static ?string $slug = 'custom-url-slug';`

My custom page is inside **Resource**. The above code is working in `Page` but not inside `Resource/Page/Custom`

**Comment by danharrin** [comment, marked as answer]
Try the `$title` property on that page

**Reply by valpuia** [reply]
Yes, this line changed. but another problem come

`protected static ?string $title = 'Custom Page Title';` this line can change breadcumb

but I want it like `protected static ?string $title = __('message.custom');` but this line throw error

so I want to override title like below, but nothing changed.

```
protected function getPageTitle()
{
     return 'abc'; //__('message.custom')
}
```

Any suggesstion to use translation text @danharrin

**Reply by danharrin** [reply]
It needs to be `getTitle()` instead of `getPageTitle()` if you want a dynamic title. Use your IDE to make sure the method is overridden correctly.

**Reply by valpuia** [reply]
Yes correct.. I try that before but it always give me error so I changed it to `getPageTitle()` but now I used like below and it's working.. thank you very much

```
protected function getTitle(): string
{
    return __('message.detail');
}
```
